index.js

```diff
@@ -184,6 +207,7 @@ class Tonic {
   _disconnect (index) {
     this.disconnected && this.disconnected()
     delete Tonic._data[this.root._id]
+    delete Tonic._elements[this.root._id]
```

Comment:
It would be awesome to create some kind of benchmarks for performance and for memory usage. Right now I'm just operating on the basis of perceived performance and some basic sanity checks for memory consumption. With non-trivial and highly complex user interfaces things feel fast, small memory footprint. But, it would be nice to have some kind of numbers.